native/src/main/java/io/ballerina/stdlib/graphql/runtime/schema/TypeFinder.java

```diff
@@ -141,6 +142,9 @@ private void getInputObjectSchemaType(Type type) {
                 int tag = field.getFieldType().getTag();
                 if (tag == TypeTags.RECORD_TYPE_TAG) {
                     getInputObjectSchemaType(field.getFieldType());
+                } else if (tag == TypeTags.STREAM_TAG) {
+                    String name = STREAM;
```

Comment:
Why do we have the `STREAM_TAG` here? It should not be a part of a GraphQL field, right?